src/UrlBarSynch.jsx

```diff
@@ -0,0 +1,62 @@
+const queryString = require('query-string');
```

Comment:
Yeah - makes sense to keep it if we think it will use the component lifecycle or need to interact with the component methods at some point. Can always change it if we find we don't need that stuff and keep it simple for now!